src/components/ZclCreateModifyEndpoint.vue

```diff
@@ -544,9 +544,11 @@ export default {
       }
     },
     getEndpointIds() {
-      this.$store.dispatch('zap/getEndpointIds').then((res) => {
-        this.endpointIds = res.data
-      })
+      if (this.$serverGet != null) {
```

Comment:
When is `this.$serverGet ` null? What is the reason for this change?

Reply:
`ui.test.js` call `mount` and `shallowMount` functions to simulate creating frontend components and in this case `$serverGet` is null. This check is very common in frontend files